appveyor.yml

```diff
@@ -8,8 +8,8 @@ environment:
   TOX_ENV: "pywin"
 
   matrix:
-    - PYTHON: "C:\\Python27"
-      PYTHON_VERSION: "2.7.8"
+    - PYTHON: "C:\\Python35"
+      PYTHON_VERSION: "3.5.2"
```

Comment:
ok, noted. Are you able to update the installation docs for windows with this information?